### PR TITLE
1836 editorial

### PR DESCRIPTION
**Pull request by azaroth42**
Make lots of little editorial changes, per #1836

### CODE REVIEW

File: source/api/presentation/3.0/index.md [hunk 1]
```diff
@@ -376,11 +374,11 @@ The value _MUST_ be an [XSD dateTime literal][org-w3c-xsd-datetime]. The value _
...
 
 Clients _MAY_ display the content of a linked placeholder Canvas when presenting the resource. When more than one such Canvas is available, for example if `placeholderCanvas` is provided for the currently selected Range and the current Manifest, the client _SHOULD_ pick the one most specific to the content. Publishers _SHOULD NOT_ assume that the placeholder Canvas will be processed by all clients. Clients _SHOULD_ take care to avoid conflicts between time-based media in the rendered placeholder Canvas and the content of the resource that has the `placeholderCanvas` property.
 
-The value _MUST_ be a JSON object with the `id` and `type` properties, and _MAY_ have other properties of Canvases. The object _MUST NOT_ have the `placeholderCanvas` property, nor the `accompanyingCanvas` property.
+The value _MUST_ be a JSON object with the `id` and `type` properties, and _MAY_ have other properties of Canvases. The value of `type` _MUST_ be "Canvas". The object _MUST NOT_ have the `placeholderCanvas` property, nor the `accompanyingCanvas` property.
```
Comment (zimeon) [inline source/api/presentation/3.0/index.md]: `Canvas` should be in back-ticks not double quotes

File: source/api/presentation/3.0/index.md [hunk 2]
```diff
@@ -410,7 +408,7 @@ A single Canvas that provides additional content for use while rendering the res
 
 Clients _MAY_ display the content of an accompanying Canvas when presenting the resource. As with `placeholderCanvas` above, when more than one accompanying Canvas is available, the client _SHOULD_ pick the one most specific to the content. Publishers _SHOULD NOT_ assume that the accompanying Canvas will be processed by all clients. Clients _SHOULD_ take care to avoid conflicts between time-based media in the accompanying Canvas and the content of the resource that has the `accompanyingCanvas` property.
 
-The value _MUST_ be a JSON object with the `id` and `type` properties, and _MAY_ have other properties of Canvases. The object _MUST NOT_ have the `placeholderCanvas` property, nor the `accompanyingCanvas` property.
+The value _MUST_ be a JSON object with the `id` and `type` properties, and _MAY_ have other properties of Canvases. The value of `type` _MUST_ be "Canvas". The object _MUST NOT_ have the `placeholderCanvas` property, nor the `accompanyingCanvas` property.
```
Comment (zimeon) [inline source/api/presentation/3.0/index.md]: Again, `Canvas` should be in back-ticks not double quotes

File: source/api/presentation/3.0/index.md [hunk 3]
```diff
@@ -483,15 +481,15 @@ The value _MUST_ be a string.
 
 The specific media type (often called a MIME type) for a content resource, for example `image/jpeg`. This is important for distinguishing different formats of the same overall type of resource, such as distinguishing text in XML from plain text.
 
+Please note that this is different to the `formats` property in the [Image API][image-api], which gives the extension to use within that API. It would be inappropriate to use in this case, as `format` can be used with any content resource, not just images.
```
Comment (zimeon) [inline source/api/presentation/3.0/index.md]: I would be inclined to get rid of the "Please", and start "Note that..." -- but don't feel very strongly on the issue

File: source/api/presentation/3.0/index.md [hunk 4]
```diff
@@ -105,12 +103,12 @@ This specification defines the following resource types:
 ##### Collection
 {: #overview-collection}
 
-An ordered list of Manifests, and/or further Collections. Collections allow easy navigation among the Manifests in a hierarchical structure, potentially each with its own descriptive information. Collections might be used to model dynamic result sets from a search, fixed sets of related resources, or other groupings of Manifests for presentation.
+An ordered list of Manifests, and/or further Collections. Collections allow Manifests and child Collections to be grouped in a hierarchical structure for presentation, which can be for of generating navigation, showing dynamic results from a search, or providing fixed sets of related resources for any other purpose.
```
Comment (mikeapp) [inline source/api/presentation/3.0/index.md]: "can be for of" - strike "of" or rephrase...

**Pull request merged**